### PR TITLE
Add instructor training video screenshot links

### PR DESCRIPTION
**Pull request by trevordmiller**
- Add video screenshot links to relevant sections
- Update .gitignore to remove more generated files/dirs

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
-node_modules
-.idea/
\ No newline at end of file
+_book/
```
Comment (trevordmiller) [inline .gitignore]: The last time I committed to this repo the build had to be committed, but I noticed it was removed a bit ago so I assume this gets built with continuous integration?

File: 02-creating-lessons/recording-the-screen.md [hunk 2]
```diff
@@ -160,4 +172,8 @@ You want to avoid a situation where you are doing intense editing to move audio
 
 You do want to go back and make sure you edit out mouth noises and excessive "um"s. If you find yourself having to edit these out, it might take some pre-thought to avoid including them in the first place.
 
-:zap: **the easiest way to edit is to capture it well **
+:zap: **the easiest way to edit is to capture it well**
+
+This video shows the result of an egghead style lesson after editing:
+
+[![Final result](screenshots/final-result.png)](https://egghead.io/lessons/tools-review-final-screencast-result)
```
Comment (trevordmiller) [inline 02-creating-lessons/recording-the-screen.md]: We might want to think about breaking up this page as it is pretty lengthy; it covers preparing to record, recording, and editing.

File: 02-creating-lessons/egghead-lesson.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 ## What's an egghead lesson?
 
+This video will show you the differences between egghead and non-egghead style screencasts:
+
+[![egghead style](screenshots/egghead-style.png)](https://egghead.io/lessons/tools-create-a-self-contained-code-focused-screencast)
+
```
Comment (trevordmiller) [inline 02-creating-lessons/egghead-lesson.md]: I didn't quite know where to put this; it has overlapping information in chapter 1 as well as the first three files of chapter 2. This seemed to make the most sense for now, but we may want to consolidate chapter 1 and the first three files of chapter 2.

Comment (joelhooks) [inline 02-creating-lessons/egghead-lesson.md]: Chapter 2 is probably appropriate

File: 02-creating-lessons/recording-the-screen.md [hunk 4]
```diff
@@ -140,6 +148,10 @@ It can be challenging to sync and get a natural feeling with narration. Like mos
 
 #### Editing
 
+This video will show you how you can can edit with ripple delete:
```
Comment (joelhooks) [inline 02-creating-lessons/recording-the-screen.md]: I'd move this after the text paragraph that follows. Read then watch. That's true for a few of these.

**Comment by joelhooks** [comment]
These will be a nice addition 👍

File: 02-creating-lessons/recording-the-screen.md (hunk 4 above)
Comment (trevordmiller) [inline 02-creating-lessons/recording-the-screen.md]: Yeah no problem will do

File: 02-creating-lessons/recording-gear.md [hunk 5]
```diff
@@ -65,3 +65,10 @@ XLR is a type of plug that is used in professional audio applications.
 
 > **TODO** Some videos for gear setup?
 
+#### Check Audio Input Levels
+
+Once your gear is set up, you'll need to check the audio input levels to make sure your gain is set properly.
```
Comment (trevordmiller) [inline 02-creating-lessons/recording-gear.md]: Added this line for a little intro text like the others

**Comment by trevordmiller** [comment]
I've shifted the videos below text; I added one line of text for this to make more sense in one case.

**Pull request merged**